src/net_processing.cpp

```diff
@@ -1528,6 +1528,7 @@ void RelayTransaction(const uint256& txid, const uint256& wtxid, const CConnman&
 
 static void RelayAddress(const CAddress& addr, bool fReachable, const CConnman& connman)
 {
+    if (!fReachable && !addr.IsRelayable()) return;
```

Comment:
The following code https://github.com/bitcoin/bitcoin/blob/56f9dba015c592b8925795012e3061a710070a27/src/net_processing.cpp#L1540-L1541

does not work for `!fReachable` addresses now.

At least, the comment is wrong now, right?